src/lib/render-components.js

```diff
@@ -1,29 +1,40 @@
 // Libraries
 import React, { Fragment } from 'react'
 
+// Lib
+import Config from './config'
+
 // Components
 import { buildTemplateComponentsManifest } from '@shiftcommerce/shift-react-components'
 
-export default function renderComponents (componentsData) {
-  let components = []
-  const templateComponentsManifest = buildTemplateComponentsManifest()
+/**
+ * Get the relevant templates for the page components
+ * @param  {Array}    pageComponents
+ * @return {String} - HTML markup for the component
+ */
+export default function renderComponents (pageComponents) {
+  // Merge any custom template components with the ones from shift-react-components
+  const templateComponentsManifest = Object.assign({}, buildTemplateComponentsManifest(), Config.get().customTemplateComponents)
```

Comment:
With this change in place shift-react-components can now export the templateComponentManifest directly instead of exporting `buildTemplateComponentsManifest` . The reason the manifest was exported as a function was so that the call to config happens at runtime.

Reply:
I'll make a ticket for this